relay-config/src/config.rs

```diff
@@ -2003,6 +2004,80 @@ impl Config {
         self.values.processing.max_rate_limit.map(u32::into)
     }
 
+    /// Creates an [`AggregatorConfig`] that is compatible with every other aggregator.
+    ///
+    /// A lossless aggregator can be put in front of any of the configured aggregators without losing data that the configured aggregator would keep.
+    /// This is useful for pre-aggregating metrics together in a single aggregator instance.
+    pub fn permissive_aggregator_config(&self) -> AggregatorConfig {
+        let aggregators: Vec<AggregatorConfig> = self
+            .values
+            .secondary_aggregators
+            .iter()
+            .map(|agg| &agg.config.aggregator)
+            .chain(std::iter::once(&self.values.aggregator.aggregator))
+            .cloned()
+            .collect();
+
+        // The unwraps are safe, as it's only 'None' if the vector is empty.
```

Comment:
Please, do not use unwraps, even though they are safe now.
It's better to be defensive in the beginning and make sure the future code, if it's about to change, won't fail because of these unwraps. 

I would suggest to fallback to defaults for now.

Reply:
I followed joris approach now, which is without unwraps